subprojects/docs/src/docs/userguide/authoring-builds/incremental_builds.adoc

```diff
@@ -394,15 +400,13 @@ There are other task types where this kind of nested inputs are available:
 In the same way, this kind of modelling is available to custom tasks.
 
 [[sec:task_input_validation]]
-==== Runtime validation
+=== Task input validation
 
 When executing the build Gradle checks if task types are declared with the proper annotations.
 It tries to identify problems where e.g. annotations are used on incompatible types, or on setters etc.
 Any getter not annotated with an input/output annotation is also flagged.
 These problems then fail the build or are turned into deprecation warnings when the task is executed.
 
-[NOTE]
```

Comment:
❌  I think I'd like to keep this around as a note or warning. Though I don't feel very strongly about it.

Reply:
I don't feel strongly either. The reason I removed the NOTE wrapper is because this "note" is taking up more visual estate than the actual content, and it's content feels more like the main content rather than a side note.

![image](https://user-images.githubusercontent.com/2759152/208424795-c47966b1-ced4-47fd-bc4c-44151288e8b9.png)